NDTensors/src/Unwrap/src/functions/util.jl

```diff
@@ -0,0 +1,5 @@
+Base.parent(E::Exposed) = parent(unexpose(E))
```

Comment:
Maybe we could call this file `abstractarray.jl` to indicate it is functionality focused on working with `AbstractArray` objects.